modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -12,48 +12,60 @@
 /datum/job/ai
 	alt_titles = list(
 		"AI",
+		"Automated Overseer",
 		"Station Intelligence",
-		"Automated Overseer"
 	)
 
 /datum/job/assistant
 	alt_titles = list(
 		"Assistant",
-		"Civilian",
-		"Tourist",
+		"Artist",
 		"Businessman",
 		"Businesswoman",
-		"Trader",
+		"Civilian",
 		"Entertainer",
 		"Freelancer",
-		"Artist",
-		"Off-Duty Staff",
+		"Tourist",
+		"Trader",
 		"Off-Duty Crew",
+		"Off-Duty Staff",
 	)
 
 /datum/job/atmospheric_technician
 	alt_titles = list(
 		"Atmospheric Technician",
-		"Life Support Technician",
 		"Emergency Fire Technician",
 		"Firefighter",
+		"Life Support Technician",
 	)
 
 /datum/job/barber
 	alt_titles = list(
 		"Barber",
+		"Aethestician",
+		"Colorist",
 		"Salon Manager",
 		"Salon Technician",
 		"Stylist",
-		"Colorist",
 	)
 
 /datum/job/bartender
 	alt_titles = list(
 		"Bartender",
-		"Mixologist",
-		"Barkeeper",
 		"Barista",
+		"Barkeeper",
+		"Mixologist",
+	)
+
+/datum/job/bitrunner
+	alt_titles = list(
+		"Bitrunner",
+		"Bitdomain Technician",
+		"Data Retrieval Specialist",
+		"Netdiver",
+		"Pod Jockey",
```

Comment:
Instead of Netdiver, how about Netrunner?
Also Pod Jockey is way too much of an unprofessional and irrelevant name for a role like bitrunner in my opinion.

Reply:
Netrunner is a copyright of R. Talsorian Games Inc.

Reply:
That is true, my bad. But what about Pod Jockey?

Reply:
pod jockey is poggers. i think bitrunners, given the nature of their jobs, have a bit of leeway in their titles. I mean, their official title is bitrunner. Sounds like some made up video game shit. That's why I put Data Retrieval Specialist in there as a more corporate-lingo themed title.